x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/use_columns.tsx

```diff
@@ -273,9 +332,148 @@ export const useColumns = (
         sortable: true,
         valign: 'middle',
       },
+      ['Baseline rate']: {
+        'data-test-subj': 'aiopsLogRateAnalysisResultsTableColumnBaselineRateChange',
+        field: 'bg_count',
+        name: (
+          <>
+            <FormattedMessage
+              id="xpack.aiops.logRateAnalysis.resultsTable.baselineRateLabel"
+              defaultMessage="Baseline rate"
+            />
+            &nbsp;
+            <EuiIconTip
+              size="s"
+              position="top"
+              color="subdued"
+              type="questionInCircle"
+              className="eui-alignTop"
+              content={baselineRateMessage}
+            />
+          </>
+        ),
+        render: (_, { bg_count: bgCount }) => {
+          if (
+            interval === 0 ||
+            windowParameters === undefined ||
+            buckets === undefined ||
+            isGroupsTable
+          )
+            return NOT_AVAILABLE;
+
+          const { baselineBucketRate } = getBaselineAndDeviationRates(
+            buckets.baselineBuckets,
+            buckets.deviationBuckets,
+            undefined,
+            bgCount
+          );
+
+          return <>{baselineBucketRate}</>;
+        },
+        sortable: false,
+        valign: 'middle',
+      },
+      ['Deviation rate']: {
+        'data-test-subj': 'aiopsLogRateAnalysisResultsTableColumnDeviationRateChange',
+        field: 'doc_count',
+        name: (
+          <>
+            <FormattedMessage
+              id="xpack.aiops.logRateAnalysis.resultsTable.deviationRateLabel"
+              defaultMessage="Deviation rate"
+            />
+            &nbsp;
+            <EuiIconTip
+              size="s"
+              position="top"
+              color="subdued"
+              type="questionInCircle"
+              className="eui-alignTop"
+              content={deviationRateMessage}
+            />
+          </>
+        ),
+        render: (_, { doc_count: docCount }) => {
+          if (
+            interval === 0 ||
+            windowParameters === undefined ||
+            buckets === undefined ||
+            isGroupsTable
+          )
+            return NOT_AVAILABLE;
+
+          const { deviationBucketRate } = getBaselineAndDeviationRates(
+            buckets.baselineBuckets,
+            buckets.deviationBuckets,
+            docCount,
+            undefined
+          );
+
+          return <>{deviationBucketRate}</>;
+        },
+        sortable: false,
+        valign: 'middle',
+      },
+      ['Log rate change']: {
```

Comment:
Can the `Log rate change` column be made sortable? I think the similar column in the anomalies table sorts by size of factor of increase / decrease.

Reply:
Updated in ff861eb3f280d3812b7536fcbabcd06eced039cc
Since we have the pValue - which is an indication of the impact - I used that to sort 👍

Reply:
Dosen't look like `pValue` always correlates to the size of the change.

<img width="727" alt="Screenshot 2024-06-21 at 11 54 38" src="https://github.com/elastic/kibana/assets/7405507/d6f47d68-0f60-4a42-9abf-93e241727c19">


I think the `sortable` prop can be `((item: T) => Primitive)` as well as a simple boolean and then you can use logic like https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/server/models/results_service/build_anomaly_table_items.js#L105 to return the factor difference (or maybe you already know the change up front?).